src/pages/Search/SearchMoneyRequestReportHoldReasonPage.tsx

```diff
@@ -16,12 +16,13 @@ import INPUT_IDS from '@src/types/form/MoneyRequestHoldReasonForm';
 function SearchMoneyRequestReportHoldReasonPage({route}: PlatformStackScreenProps<SearchReportParamList, typeof SCREENS.SEARCH.TRANSACTION_HOLD_REASON_RHP>) {
     const {translate} = useLocalize();
 
-    const {backTo, reportID} = route.params;
+    const {backTo, reportID, searchHash} = route.params;
     const {selectedTransactionsID, setSelectedTransactionsID} = useMoneyRequestReportContext();
 
     const onSubmit = (values: FormOnyxValues<typeof ONYXKEYS.FORMS.MONEY_REQUEST_HOLD_FORM>) => {
-        selectedTransactionsID.forEach((transactionID) => putOnHold(transactionID, values.comment, reportID));
-
+        if (selectedTransactionsID.length) {
```

Comment:
Not your code, but let's rename this so it's more readble

```suggestion
        if (selectedTransactionIDs.length) {
```

Reply:
Hmm, the shorthand in `MoneyRequestReportContext` is `selectedTransactionsID`. Should I rename it to `selectedTransactionIDs` in `SearchMoneyRequestReportHoldReasonPage` alone? 

https://github.com/Expensify/App/blob/9a3e5308009d8e0bed18b26d885393d8324f263b/src/components/MoneyRequestReportView/MoneyRequestReportContext.tsx#L49